tests/operator/operator.go

```diff
@@ -2444,7 +2456,8 @@ spec:
 			}, 240*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
 
 			// wait for virt-api and virt-controller to pick up the change that CDI no longer exists.
-			time.Sleep(30 * time.Second)
+			time.Sleep(5 * time.Second)
```

Comment:
Is sleep enough?

Reply:
Nope, because there is a 5 sec sleep before the virt API terminates